### DISCUSSION TITLE
OP_66 - manual cursed inscriptions bound

### DISCUSSION THREAD

**Discussion by dannydeezy** [ordinals/ord#2113] [General]
Following up from https://github.com/casey/ord/pull/2109#issuecomment-1562340598

Many users have inscribed with OP_66 as a way to manually trigger "cursed" inscriptions. How do we feel about adding this as an op code that would allow the cursed inscription to be properly bound to the expected sat?

**Comment by dannydeezy** [comment]
one thought is to deny `OP_66` to set a precedent for degens to not speculate and front-run things based on github discussions (https://github.com/casey/ord/pull/2109#issuecomment-1562326574)

**Reply by DrJingLee** [reply]
Can feel the excitement wen ppl realized there is a “cursed”inscriptions...PPL are expecting another 10m "cursed"inscriptions...But From the previous discussion seems the OP_66 is not transferable ...So will this like SBT ? Or a fork of ord not support it in further development/upgrades .

**Reply by dannydeezy** [reply]
the discussion is ongoing if OP_66 will be recognized as transferrable or not, and it also depends on the decision to make all unknown even tagged inscriptions untransferable (https://github.com/casey/ord/pull/2109)

**Reply by tyjvazum** [reply]
Inscriptions with unknown even tags have always been nontransferable (effectively not considered inscriptions) in `ord`: https://github.com/casey/ord/pull/2109#issuecomment-1562180592

**Comment by spencerrichardhenry** [comment]
It seems silly to not re-index for cursed ordinals originially because it would offset some ordinals by like 100 or something, but then throw out thousands of ordinals because of a change to op-code recognition. The project should be consistent in its judgment of impact IMO

**Reply by GMFjoe** [reply]
🐐

**Comment by shep-eth** [comment]
Would appreciate OP_66 recognition by any chance

**Comment by drahlraigon** [comment]
I personally think the lesson has been learned, people got in the trend and inscribed negative inscriptions deliberately based on github conversations. I think if we discontinue support for OP_66 it would look against what ordinals and BTC truly stands for, decentralisation. Think it is only fair to let the community vote or be a part of this decision atleast. This is just a request and only what i feel, would be pretty cool if we added support for already inscribed OP_66

**Comment by ddiekroeger** [comment]
![image](https://github.com/casey/ord/assets/72826230/94f9c086-b6d7-4c81-82e3-d574b4a50085)

**Comment by tyjvazum** [comment]
I'm willing to support unrecognized even tags being transferable in [arb](https://github.com/tyjvazum/arb) by removing the current restriction without making such inscriptions unbound. This can be implemented relatively quickly. Future upgrades using even tags will still be possible based on activation height. Is anyone interested in this solution?

**Reply by dannydeezy** [reply]
i think this might sound ok, but would like input from other devs as I think it's most important we come to a consensus, and we shouldn't rush with solutions that aren't widely accepted as it will result in conflicting implementations

**Reply by tyjvazum** [reply]
I agree in general, but I unfortunately think conflicting implementations are inevitable given the lack of public planning and the fact that the only consensus mechanism available is social. What do you see as potential issues?

**Reply by ehutzle** [reply]
> it's most important we come to a consensus

good luck 

https://forkdrop.io/how-many-bitcoin-forks-are-there

**Comment by spencerrichardhenry** [comment]
@raphjaph given your commit on the unbound issue, can we assume that op_66 cursed ordinals will be unbound for the foreseeable future? It was unclear to me whether that was a final decision, or just a separation of code commits between that issue and https://github.com/raphjaph/ord/pull/10
